internal/gribi/gribi.go

```diff
@@ -218,6 +218,14 @@ func (c *Client) AddNH(t testing.TB, nhIndex uint64, address, instance string, e
 			NH = NH.WithNextHopNetworkInstance(opt.VrfName)
 		}
 		c.fluentC.Modify().AddEntry(t, NH)
+	case "":
```

Comment:
Reserve a new keyword here, instead of ""? E.g. "VRF-ONLY"

Reply:
done